tsconfig.json

```diff
@@ -14,13 +16,13 @@
 		"strict": true,
 		"forceConsistentCasingInFileNames": true,
 		"noFallthroughCasesInSwitch": true,
-		"module": "esnext",
-		"moduleResolution": "node",
 		"resolveJsonModule": true,
 		"isolatedModules": true,
 		"noEmit": false,
 		"jsx": "react-jsx",
-		"downlevelIteration": true
+		"downlevelIteration": true,
+		"importHelpers": true
 	},
-	"include": ["src"]
+	"include": ["src"],
+	"exclude": ["src/**/*.stories.tsx"]
```

Comment:
This exclude will now stop tsc from actually type checking the stories as well.
I think we need to create a separate tsconfig.dev.json which includes stories as well and have that run from a separate tsc statement in the tests w/ --noEmit. We will anyway have to do this once we move to the latest eslint-based balena-lint, b/c it is only able to lint files pointed by the tsconfig.

Eg:
See: https://github.com/balena-io/balena-sdk/blob/1f4b43a6a15231b090fc3c85be5eed69687408a0/package.json#L44

Reply:
nice catch @thgreasi. I can't find an easy way to point the storybook server to another file than `tsconfig.json` though, what do you think about preventing the files from getting published in a .npmignore file and removing the `exclude` rule here?